app/code/Magento/InventoryCatalog/Ui/DataProvider/Product/Listing/Modifier/Quantity.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Ui\DataProvider\Product\Listing\Modifier;
+
+use Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\AbstractModifier;
+use Magento\InventoryCatalog\Model\IsSingleSourceModeInterface;
+
+/**
+ * Quantity modifier on CatalogInventory Product Grid
+ */
+class Quantity extends AbstractModifier
+{
+
```

Comment:
Please, remove an extra line here and in other modifiers

Reply:
@rogyar done.